File: src/main/java/com/abrastat/runners/CurseSlamRestTalkSnorlax_VS_ThunderIceRestTalkZapdos.java
package com.abrastat.runners;

import com.abrastat.general.Item;
import com.abrastat.general.Messages;
import com.abrastat.general.Pokemon;
import static com.abrastat.gsc.GSCMove.*;
import com.abrastat.general.Type;
import com.abrastat.gsc.GSCPokemon;

public class CurseSlamRestTalkSnorlax_VS_ThunderIceRestTalkZapdos {

    public static void main(String[] args) {
        Pokemon snorlax = new GSCPokemon.Builder("snorlax")
                .moves(BODY_SLAM, CURSE, REST, SLEEP_TALK)
                .item(Item.LEFTOVERS)
                .build();

        Pokemon zapdos = new GSCPokemon.Builder("zapdos")
                .moves(THUNDER, HIDDEN_POWER, REST, SLEEP_TALK)
                .hiddenPowerType(Type.ICE)
                .item(Item.LEFTOVERS)
                .build();


        new GSCGameRunner(snorlax, zapdos, 100000);

        Messages.gameOver();
        System.exit(0);
    }
}

File: src/main/java/com/abrastat/runners/CurseSlamRestTalkSnorlax_VS_ThunderIceRestTalkZapdos.kt
package com.abrastat.runners

import com.abrastat.general.Item
import com.abrastat.general.Messages.Companion.gameOver
import com.abrastat.general.Type
import com.abrastat.gsc.GSCMove
import com.abrastat.gsc.GSCPokemon
import kotlin.system.exitProcess

object CurseSlamRestTalkSnorlax_VS_ThunderIceRestTalkZapdos {
    @JvmStatic
    fun main(args: Array<String>) {
        val snorlax = GSCPokemon.Builder("snorlax")
                .moves(GSCMove.BODY_SLAM, GSCMove.CURSE, GSCMove.REST, GSCMove.SLEEP_TALK)
                .item(Item.LEFTOVERS)
                .build()
        val zapdos = GSCPokemon.Builder("zapdos")
                .moves(GSCMove.THUNDER, GSCMove.HIDDEN_POWER, GSCMove.REST, GSCMove.SLEEP_TALK)
                .hiddenPowerType(Type.ICE)
                .item(Item.LEFTOVERS)
                .build()
        GSCGameRunner(snorlax, zapdos, 1)
        gameOver()
        exitProcess(0)
    }
}